### PR TITLE
certbot_instruction updates

### LINKED ISSUE #6215: fix(doc):  update certbot instruction
- [ ] update instrucdtion for ubuntu
- [ ] update instrucdtion for suse
- [ ] update instrucdtion for RHEL

### PR DESCRIPTION
**Pull request by manojs1978**
### Prepare

- [ ] Read [PR guidelines](https://github.com/JanssenProject/jans/blob/main/docs/CONTRIBUTING.md#prs)
- [ ] Read [license information](https://github.com/JanssenProject/jans/blob/main/LICENSE)

-------------------

#### Target issue
  
  
  
  
closes #6215

#### Implementation Details
  

-------------------
### Test and Document the changes
- [ ] Static code analysis has been run locally and issues have been fixed
- [ ] Relevant unit and integration tests have been added/updated
- [ ] Relevant documentation has been updated if any (i.e. user guides, installation and configuration guides, technical design docs etc)

### CODE REVIEW

**Review by mo-auto — APPROVED** [review]

File: docs/admin/install/vm-install/ubuntu.md [hunk 1]
```diff
@@ -131,6 +131,22 @@ sudo /opt/jans/jans-cli/jans_cli_tui.py
...
+sudo certbot --apache -d FQDN
+```
+to check certbot status
+```
+sudo systemctl status certbot.timer
+```
+to renew certificate run
+```
+sudo certbot renew --dry-run
+```
```
Comment (ossdhaval) [inline docs/admin/install/vm-install/ubuntu.md]: @manojs1978 For Ubuntu, don't we have to update values in `https_jans.conf` ?

File: docs/admin/install/vm-install/rhel.md [hunk 2]
```diff
@@ -102,6 +102,33 @@ sudo /opt/jans/jans-cli/jans_cli_tui.py
 
 Full TUI documentation can be found [here](../../config-guide/config-tools/jans-tui/README.md)
 
+If you have selected casa during installation you can access casa using url``` https://<host>/jans-casa ```
+
+## Let's Encrypt
```
Comment (ossdhaval) [inline docs/admin/install/vm-install/rhel.md]: This section primarily has two parts:
1) Instructions for getting certificate (from Let's encrypt)
2) Instructions for configuring certificates in Jans

I believe this document should only contain the `2` as it pertains to Jans. Let's encrypt being one of many ways of getting certificates, it shouldn't be part of install instructions. May be it should be moved to `how-to` docs and give a link here.

File: docs/admin/install/vm-install/rhel.md [hunk 3]
```diff
@@ -102,6 +102,33 @@ sudo /opt/jans/jans-cli/jans_cli_tui.py
...
+ To generate Let’s Encrypt CA certificate run below commands:
+```
```
Comment (ossdhaval) [inline docs/admin/install/vm-install/rhel.md]: Also we need to add some context about `why certificates are needed`

**Review by ossdhaval — CHANGES_REQUESTED** [review]

File: docs/admin/install/vm-install/ubuntu.md (hunk 1 above)
Comment (manojs1978) [reply docs/admin/install/vm-install/ubuntu.md]: no it will automatically done

**Review by mo-auto — DISMISSED** [review]

File: docs/admin/install/vm-install/rhel.md (hunk 2 above)
Comment (manojs1978) [reply docs/admin/install/vm-install/rhel.md]: created how-to doc https://github.com/JanssenProject/jans/pull/6323
and kept only jans related instruction ie  point 2

**Review by mo-auto — APPROVED** [review]

**Review by ossdhaval — APPROVED** [review]

**Pull request merged**